source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md

```diff
@@ -17,6 +17,660 @@ including minor and bugfix releases, refer to the `UPGRADE.md` file found in you
+* Added entity `Shopware\Models\Customer\CustomerStream` for attribute single and multi selection.
+* Added translations for attribute labels. See below for more information.
+* Added database structure for new emotion preset feature:
+    * `s_emotion_presets` - contains all installed presets
+    * `s_emotion_preset_translations` - contains presets translations
+* Added models for presets
+    * `Shopware\Models\Emotion\Preset`
+    * `Shopware\Models\Emotion\PresetTranslation`
+* Added classes for handling emotion preset feature
+    * `Shopware\Components\Emotion\EmotionImporter` - handle emotion imports
+    * `Shopware\Components\Emotion\EmotionExporter` - handle emotion exports
+    * `Shopware\Components\Emotion\Preset\EmotionToPresetDataTransformer` - transform emotion to preset
+    * `Shopware\Components\Emotion\Preset\PresetDataSynchronizer` - uses component handlers to support import / export of emotions  
+    * `Shopware\Components\Emotion\Preset\PresetInstaller` - installer for preset plugins
+    * `Shopware\Components\Emotion\Preset\PresetLoader` - loads presets and refreshes preset data to match current database
+    * `Shopware\Components\Emotion\Preset\PresetMetaDataInterface` - interface to use for preset plugin development
+* Added API Resource for emotion presets `Shopware\Components\Api\Resource\EmotionPreset`
+* Added backend controller for emotion presets `Shopware\Controllers\Backend\EmotionPresets`
+* Added compiler pass to register emotion component handlers `Shopware\Components\DependencyInjection\Compiler\EmotionPresetCompilerPass`
+* Added component handlers for asset import and export of shopping world elements
+    * `Shopware\Components\Emotion\Preset\ComponentHandler\BannderComponentHandler`
+    * `Shopware\Components\Emotion\Preset\ComponentHandler\BannerSliderComponentHandler`
+    * `Shopware\Components\Emotion\Preset\ComponentHandler\CategoryTeaserComponentHandler`
+    * `Shopware\Components\Emotion\Preset\ComponentHandler\Html5VideoComponentHandler`
+* Added new ExtJs views for emotion presets under `themes\backend\emotion\view\preset`
+* Added new service tag for registering emotion preset component handlers `shopware.emotion.preset_component_handler`
+* Added actions to import and export shopping worlds in `Shopware_Controllers_Backend_Emotion`
+* Changed theme path for new plugins from `/resources` into `/Resources`
+* Changed sorting of `Shopware.listing.FilterPanel` fields
+* Changed database column `s_articles_vote`.`answer_date` to allow `NULL` values
+* Changed `LastArticle` plugin config elements `show`, `controller` and `time` to be prefixed with `lastarticles_`
+* Changed product listings in shopping worlds to only be loaded if `showListing` is true
+* Changed sql query in `sAdmin` queries which uses a sub query for address compatibility, following functions affected:
```

Comment:
@jenskueper These are snippets from the upgrade.md. But this is a good catch. We have to evaluate which are necessary and which maybe can be left out.